docs/operator-manual/config-management-plugins.md

```diff
@@ -19,9 +18,10 @@ The following sections will describe how to create, install, and use plugins. Ch
 
 ## Installing a config management plugin
 
-### Sidecar plugin
+### Sidecar or services plugin
 
-An operator can configure a plugin tool via a sidecar to repo-server. The following changes are required to configure a new plugin:
+An operator can configure a plugin tool via a sidecar to the repo-server, or as a Kubernetes service in the same cluster
+as Argo CD. The following changes are required to configure a new plugin:
```

Comment:
```suggestion
An operator can configure a plugin tool as a repo-server sidecar or as a Kubernetes service. The following changes are required to configure a new plugin:
```

I think the cluster constraint is something people would assume. (And notably, the actual thing doesn't have to be in the same cluster as you note by `ExternalName` below.)